evaluation/dp_verification.py

```diff
@@ -83,7 +83,7 @@ def generate_neighbors(self, load_csv = False):
         d1_table.schema, d2_table.schema = "d1", "d2"
         d1_table.name, d2_table.name = "d1", "d2"
         d2_table.rowcount = d1_table.rowcount - 1
-        d1_metadata, d2_metadata = Collection([d1_table], "csv"), Collection([d2_table], "csv")
```

Comment:
the import change?

Reply:
Yes, that code works; the import is line 20